osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/smartmetering/ActualPowerQualityRequest.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2021 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance 
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.domain.core.valueobjects.smartmetering;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import org.opensmartgridplatform.domain.core.valueobjects.DeviceFunction;
+import org.opensmartgridplatform.shared.exceptionhandling.FunctionalException;
+
+public class ActualPowerQualityRequest implements Serializable, ActionRequest {
```

Comment:
ActionRequest is already a Serializable, the Serializable interface here can be removed.

Reply:
Serializable removed from implements